src/html5/parser/quirks.rs

```diff
@@ -156,15 +156,13 @@ static LIMITED_QUIRKS_PUB_IDENTIFIER_PREFIX_NOT_MISSING_SYS: &[&str] = &[
 #[cfg(test)]
 mod tests {
     use crate::html5::input_stream::InputStream;
-    use crate::html5::parser::document::DocumentBuilder;
     use crate::html5::parser::Html5Parser;
     use crate::html5::parser::QuirksMode;
 
     #[test]
     fn test_quirks_mode() {
-        let doc = DocumentBuilder::new_document();
-        let mut stream = InputStream::new();
-        let parser = Html5Parser::new(&mut stream, doc.clone());
+        let stream = &mut InputStream::new();
```

Comment:
During testing, we need to have a parser instance that doesn't care about the stream and document it contains. In these cases we can use `Html5Parser::new_parser()`. I doubt this function is called anywhere but in unit tests.